src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -34,8 +34,8 @@ export interface UserProfilePictureProp {
 // @debt the UserAvatar component serves a very similar purpose to this, we should unify them as much as possible
 const UserProfilePicture: React.FC<UserProfilePictureProp> = ({
   isAudioEffectDisabled,
```

Comment:
You seem to have lost the default on this that was the original fix in this PR. That’s still required